packages/web3-constants/evm/ito.json

```diff
@@ -158,6 +164,7 @@
         "Aurora_Testnet": "",
         "Conflux": "",
         "Harmony": "https://api.thegraph.com/subgraphs/name/dimensiondev/mask-ito-harmony",
-        "Harmony_Test": ""
+        "Harmony_Test": "",
+        "Astar": "https://api.thegraph.com/subgraphs/name/dimensiondev/mask-ito-astar"
```

Comment:
We don't deployed subgraph for Astar yet, nor Harmony.

<img width="1105" alt="image" src="https://user-images.githubusercontent.com/1141198/179438752-747b763b-feef-4f53-a1b8-70b8fcbfdf5a.png">

But I think you can keep this config.

cc @guanbinrui